src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheck.java

```diff
@@ -29,59 +29,130 @@
 import com.puppycrawl.tools.checkstyle.api.DetailAST;
 
 /**
- * Checks correct indentation of Java Code.
- *
  * <p>
- * The basic idea behind this is that while
- * pretty printers are sometimes convenient for reformatting of
+ * Checks correct indentation of Java code.
+ * </p>
+ * <p>
+ * The idea behind this is that while
+ * pretty printers are sometimes convenient for bulk reformats of
  * legacy code, they often either aren't configurable enough or
- * just can't anticipate how format should be done.  Sometimes this is
- * personal preference, other times it is practical experience.  In any
+ * just can't anticipate how format should be done. Sometimes this is
+ * personal preference, other times it is practical experience. In any
  * case, this check should just ensure that a minimal set of indentation
- * rules are followed.
+ * rules is followed.
```

Comment:
@rnveach please pay attention to this change

Reply:
@romani is correct. It should remain as `are`. The subject, "indentation rules" is plural so the verb should be the plural form

Reply:
Done. I thought the subject is a *set* of rules.

Reply:
@pbludov You are correct.
https://parts-of-speech.info/
noun = set, indentation, rules
preposition = of

http://www.chompchomp.com/terms/prepositionalphrase.htm
> Sometimes a noun within the prepositional phrase seems the logical subject of a verb. Don't fall for that trick! You will never find a subject in a prepositional phrase. Look at this example:
> > Neither of these cookbooks contains the recipe for Manhattan-style squid eyeball stew.

> Cookbooks do indeed contain recipes. In this sentence, however, cookbooks is part of the prepositional phrase of these cookbooks. Neither—whatever a neither is—is the subject for the verb contains.

The example seems to closely resembler our sentence.
Since the singular 'set' is the subject, the verb 'is/are' should also be singular. So it should be `is`.
Sorry.

Reply:
reverted back to `is`